appveyor.yml

```diff
@@ -1,16 +1,31 @@
 image:
   - Visual Studio 2017
 clone_folder: c:\projects\source
-build_script:
-- cmd: >-
-    mkdir build
 
-    cd build
-    
-    cmake c:\projects\source -G "Visual Studio 15"
-    
-    cmake --build . --config "Debug"
+environment:
+  matrix:
+    - CMAKE_FLAGS: "-DENABLE_COVERAGE:BOOL=TRUE"
+    - CMAKE_FLAGS: "-DENABLE_COVERAGE:BOOL=TRUE -DCPP_STARTER_USE_GOOGLE_TEST=ON"
+    - CMAKE_FLAGS: "-DENABLE_COVERAGE:BOOL=TRUE -DCPP_STARTER_USE_CATCH2=ON"
+
+before_build:
+  - git clone https://github.com/catchorg/Catch2.git
+  - mv Catch2 _Catch2
+  - cd _Catch2
+  - cmake -Bbuild -H. -DBUILD_TESTING=OFF
+  - cmake --build build/              #--target install # not on Windows!
```

Comment:
--target install needs to be included, this is where it puts the headers in a user folder and exports the library so that cmake's find_package will find it.

Reply:
The install will fail without sudo/admin privileges.

Reply:
I think appveyor does give you admin priveleges. When I used target --install before, the Appveyor VM made a directory in Program Files called Catch2. CMake thinks it can find Catch2 at that point.  However, the compiler could not.

Reply:
Hey, it passed! Thanks for the help!